.github/workflows/create-sprint-board.yml

```diff
@@ -2,7 +2,7 @@
 name: "Create sprint board"
 on:
   schedule:
-    - cron: '0 2 * * 3'
+    - cron: '0 10 * * 2'
```

Comment:
I am unclear how this runs every other week. Since this was already working with a very similar call I will assume that every other week is managed elsewhere. With that assumption, I would say this looks good.